op-node/rollup/derive/l1_traversal_test.go

```diff
@@ -87,16 +110,31 @@ func TestL1TraversalAdvance(t *testing.T) {
 			fetcherErr:  errors.New("interrupted connection"),
 			expectedErr: ErrTemporary,
 		},
+		// TODO: add tests that cover the receipts to config data updates
 	}
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			src := &testutils.MockL1Source{}
 			src.ExpectL1BlockRefByNumber(test.startBlock.Number+1, test.nextBlock, test.fetcherErr)
+			info := &testutils.MockBlockInfo{
+				InfoHash:       test.nextBlock.Hash,
+				InfoParentHash: test.nextBlock.ParentHash,
+				InfoNum:        test.nextBlock.Number,
+				InfoTime:       test.nextBlock.Time,
+				// TODO: don't need full L1 info in receipts fetching API maybe?
```

Comment:
Is this todo something you would like to address?

Reply:
Opened ENG-2964, it's low priority.